src/Config.js

```diff
@@ -37,11 +37,12 @@ export class Config {
 
     // Create a new DatabaseController per request
     if (cacheInfo.databaseController) {
-      const schemaCache = new SchemaCache(cacheInfo.cacheController, cacheInfo.schemaCacheTTL);
+      const schemaCache = new SchemaCache(cacheInfo.cacheController, cacheInfo.schemaCacheTTL, cacheInfo.disablePerRequestSchemaCache);
```

Comment:
I think the constructor for schemaCache should ultimately take an options object like:

```js
new SchemaCache({
  cacheController,
  schemaCacheTTL,
  perRequestSchemaCache,
});
```

 Not sure if we should change it in this request though.